tests/conditional_compilation/test_collect.py

```diff
@@ -25,8 +25,9 @@ def test_cc_collect(test_id, model, sea_runtool, benchmark_app, collector_dir, a
         [
             sys.executable,
             str(sea_runtool),
-            f"-o={out}",
+            f"--output={out}",
```

Comment:
@asomsiko what's the reason to extend key here? Should we update script use in other repos?

Reply:
Updated for consistence - other runtool options are in long format